src/main/views/live-status.njk

```diff
@@ -0,0 +1,84 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ courtName + " Live Cases Updates" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: text,
+        href: '#',
+        attributes: {
+            onClick: "history.back();"
+        }
+    }) }}
+{% endblock %}
+
+{% block content %}
+    <h2 class="govuk-heading-l">Live hearing updates - daily court list</h2>
+    <h3 class="govuk-heading-m">{{ courtName }}</h3>
+    <p class=" govuk-body-lead">{{ updateDateTime }}</p>
+    <p class="govuk-body">The status of each hearing will be updated as events happen live in court</p>
+    {% set cases = [] %}
+    {% for item in liveCases %}
+        {% if item.event.eventStatus %}
+            {% set status = item.event.eventStatus +
+                ' - <br /><a class="govuk-link" href="status-description#' +
+                item.event.eventId +
+                '">' +
+                item.event.eventName +
+                '</a> - '
+                + item.event.eventTime
+            %}
+        {% else %}
+            {% set status = '<p class="govuk-!-font-weight-bold no-information-message">- No Information To Display</p>'  %}
+        {% endif %}
+        {% set _ = cases.push([
+            {
+                text: item.courtRoom
+            },
+            {
+                text: item.caseNumber
+            },
+            {
+                text: item.caseName
+            },
+            {
+                html: status
+            }
+        ]) %}
+    {% endfor %}
+    {{ govukTable({
+        firstCellIsHeader: false,
+        classes: 'govuk-!-margin-bottom-9',
+        head: [
+            {
+                text: 'Court number',
+                classes: 'table-column-court-number'
+            },
+            {
+                text: 'Case number',
+                classes: 'govuk-!-width-one-quarter table-header-bottom-align'
+            },
+            {
+                text: 'Case name',
+                classes: 'govuk-!-width-one-quarter table-header-bottom-align'
+            },
+            {
+                text: 'Status',
+                classes: 'table-header-bottom-align'
+            }
+        ],
+        rows: cases
+    }) }}
+    <hr class="govuk-header__container--full-width"/>
+    <a href="/view-option" class="govuk-link govuk-!-font-weight-bold">Find a court or tribunal list</a>
+    <script>
+      const timer = {{ refreshTimer }}

```

Comment:
is there any way we can add this kinda logic to the server side? if a user doesnt enable javascript then this wont work, which is fine for the resubmit stuff but this is the core functionality of the page